File: app/src/main/java/com/julio/uiandnav/MainActivity.java
package com.julio.uiandnav;

import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.Menu;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ActionBar actionBar = getSupportActionBar(); // atau getActionBar();
        actionBar.setTitle("Title");

        actionBar.setDisplayShowHomeEnabled(true);
        actionBar.setIcon(R.drawable.ic_menu); // untuk menampilkan ikon, atau
        actionBar.setLogo(R.mipmap.ic_launcher); // untuk menampilkan gambar
        actionBar.setDisplayUseLogoEnabled(true);

    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(R.menu.menu_main, menu);
        return true;
    }
}

File: app/src/main/java/com/julio/uiandnav/MainActivity.kt
package com.julio.uiandnav

import android.os.Bundle
import android.view.Menu
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        title = "Title"
        actionBar?.setDisplayHomeAsUpEnabled(true)
        actionBar?.setIcon(R.drawable.ic_menu) // untuk menampilkan ikon, atau
        actionBar?.setLogo(R.drawable.ic_launcher_foreground) // untuk menampilkan gambar
        actionBar?.setDisplayUseLogoEnabled(true)
    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        menuInflater.inflate(R.menu.menu_main, menu)
        return true
    }
}